src/main/java/domain/card/Card.java

```diff
@@ -0,0 +1,46 @@
+package domain.card;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Card {
+    private static final List<Card> CARDS;
+    static {
+        CARDS = Arrays.stream(CardType.values())
+                .flatMap(cardType -> Arrays.stream(CardNumber.values())
+                        .map(cardNumber -> new Card(cardType, cardNumber)))
+                .collect(Collectors.toUnmodifiableList());
+    }
```

Comment:
Card 캐싱 좋습니다 👍 

```suggestion
    private static final List<Card> CARDS;
    
    static {
        CARDS = Arrays.stream(CardType.values())
                .flatMap(cardType -> Arrays.stream(CardNumber.values())
                        .map(cardNumber -> new Card(cardType, cardNumber)))
                .toList();
    }
```
이렇게 변경해도 unmodifiableList가 반환됩니다. 줄바꿈도 신경써주시구요~

코딩 컨벤션은 글로 따지면 맞춤법, 띄어쓰기, 문단 나누기 등으로 볼 수 있어요. 맞춤법이나 띄어쓰기를 좀 안 지킨다고 글 읽는 데 문제는 없지만 가독성이 떨어지죠. 일반적으로 통용되는 코딩 컨벤션을 공부해보고 작성할 때부터 의식적으로 적용해보려 노력해보면 좋을 것 같아요!! 처음엔 저도 많이 틀리고 신경도 별로 안 썼는데 하다 보니 익숙해지더라구요ㅎㅎ 점점 좋아질 거예요 👍